Week3/homework/step3-bonus.js

```diff
@@ -3,8 +3,16 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  arr.sort(function(letter1, letter2) {
+    return letter1 > letter2;
+  });
+  const unique = [];
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] !== arr[i + 1]) {
+      unique.push(arr[i]);
+    }
+  }
```

Comment:
Aahaan. This logic works fine but it seems a bit too much. Try and read about a new concept in ES6 which is called Sets and see if you can use it to make this code shorter. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

Reply:
wow, this was really easy with using sets :D thank you!